manifests/params.pp

```diff
@@ -154,6 +167,8 @@
           $server_package_name = 'openssh'
           $client_package_name = 'openssh'
           $sshd_dir = '/etc/ssh'
+          $sshd_binary = '/usr/sbin/sshd'
```

Comment:
I think we don't need to set the values here when they are already in the `Solaris` block?